### PR TITLE
Add situation example

### PR DESCRIPTION
**Pull request by jvalduvieco**
* Documentation improvement
* Details:
  - Add a situation test example as in openfisca-barcelona is the main testing process and there is no example in the country template.
* Found a missing parameter in pension.

### CODE REVIEW

**Comment by fpagnoux** [comment]
Arg, didn't see this PR, sorry @jvalduvieco, and thanks @MattiSG for asking the review.

Will have a look at it tomorrow :).

**Comment by fpagnoux** [comment]
Sorry, I'm running out of time at the moment, but I swear I haven't forgotten about this PR :)

File: openfisca_country_template/tests/situations/a_single_parent_with_a_child.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    - parents: ["parent1"]
+      children: ["child1"]
+      accommodation_size: 100
+  persons:
+   - id: "parent1"
+     birth: '1961-01-15'
+     salary:
+      2017-01: 2400
+   - id: "child1"
+     birth: '2002-01-15'
```
Comment (MattiSG) [inline openfisca_country_template/tests/situations/a_single_parent_with_a_child.yaml]: As we want to give best practice examples, it seems important to me that we use a coherent style of quotes. If possible, no quotes at all would be the best if YAML does support it 🙂

File: openfisca_country_template/tests/situations/a_single_parent_with_a_child.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  households:
+    - parents: ["parent1"]
+      children: ["child1"]
+      accommodation_size: 100
+  persons:
+   - id: "parent1"
+     birth: '1961-01-15'
+     salary:
+      2017-01: 2400
+   - id: "child1"
```
Comment (MattiSG) [inline openfisca_country_template/tests/situations/a_single_parent_with_a_child.yaml]: Best practice: give real names in examples.

File: openfisca_country_template/tests/situations/a_single_parent_with_a_child.yaml [hunk 3]
```diff
@@ -0,0 +1,21 @@
+- name: "A single parent with a child situation"
+  description: A simple situation just to test
```
Comment (MattiSG) [inline openfisca_country_template/tests/situations/a_single_parent_with_a_child.yaml]: As a best practice example, we should be as clear and literate as possible (or not add a `description` at all). Maybe this test is about “Income tax should get properly computed across all members of a household”? 😃

File: openfisca_country_template/tests/situations/a_single_parent_with_a_child.yaml [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+  period: 2017-01
+  absolute_error_margin: 0
+  households:
+    - parents: ["parent1"]
+      children: ["child1"]
```
Comment (MattiSG) [inline openfisca_country_template/tests/situations/a_single_parent_with_a_child.yaml]: I believe best readability practice includes internal spaces: `[ parent1 ]` is more readable than `["parent1"]` :slig

**Comment by jvalduvieco** [comment]
Comments fixed. :)

**Comment by MattiSG** [comment]
Brilliant, thanks a lot @jvalduvieco!

**Pull request merged**